requirements.txt

```diff
@@ -1,17 +1,18 @@
+adal==1.2.7
 asn1crypto==0.24.0
 astroid==2.1.0
-azure-common==1.1.18
-azure-functions==1.0.0a5
-azure-functions-worker==1.0.0a6
-azure-storage-blob==1.4.0
-azure-storage-common==1.4.0
+azure-common==1.1.28
+azure-functions==1.10.1
+azure-functions-worker==1.1.9
+azure-storage-blob==12.11.0
+azure-storage-common==2.1.0
```

Comment:
This is a bit of a red flag - who has a dependency on azure-storage-common?

Reply:
@johanste I have updated by your comment.